automated/linux/kselftest/kselftest.yaml

```diff
@@ -48,10 +48,4 @@ run:
     steps:
         - cd ./automated/linux/kselftest/
         - ./kselftest.sh -t "${TESTPROG}" -s "${SKIP_INSTALL}" -u "${TESTPROG_URL}" -L "${SKIPLIST}" -S "${SKIPFILE}" -b "${BOARD}" -g "${BRANCH}" -e "${ENVIRONMENT}" -p "${KSELFTEST_PATH}"
-
-parse:
-    pattern: "^selftests:\\s(?P<test_case_id>\\S+)\\s[[](?P<result>(FAIL|PASS|SKIP))[]]"
-    fixupdict:
-        FAIL: fail
-        PASS: pass
-        SKIP: skip
+        - ../../utils/send-to-lava.sh ./output/result.txt
```

Comment:
Kselftest runs tests in sequence and when we see kernel crash at middle of the test run we loose all the results with this new parser at the end.

The older parser was run time parser for each test case.

Reply:
IIUC I could change this to perform the "send-to-lava" logic for each test?

Reply:
yes. 
Please refer this commit,
https://github.com/Linaro/test-definitions/commit/0fd41f9b0b65c17782e1a4b8cb5826b3af6eaec6#diff-e2e0fe7e0368d547b14606c0b5e209cb

Reply:
Ah, yes, okay. I understand now. I think I was looking at the rest runner and thinking it was struggle local, but if this is getting collected over the console or whatever, yes, that makes sense. I'll need to think about this a bit more, since correcting the kselftest test names is not trivial.

One question, though: it seems in many cases that the kernel's dmesg output is interleaved with test execution output which makes reading log codifying at best and at worst makes the test output parsing non-deterministic. It seems that LAVA is seeing the same interleaving, yes? I think that'll need fixing as well (i.e. something needs to be doing proper line-at-a-time interleaving of test output and dmesg output).

Thanks for the review! I'll get this series updated...